index-import.cpp

```diff
@@ -173,9 +173,12 @@ static IndexUnitWriter remapUnit(const std::unique_ptr<IndexUnitReader> &reader,
   });
 
   reader->foreachInclude([&](const IndexUnitReader::IncludeInfo &info) {
-    // Note this isn't revelevnt to Swift.
-    writer.addInclude(fileMgr.getFile(info.SourcePath), info.SourceLine,
-                      fileMgr.getFile(info.TargetPath));
+    const auto sourcePath = remapper.remap(info.SourcePath);
+    const auto targetPath = remapper.remap(info.TargetPath);
+
```

Comment:
Mind adding an `if (Verbose)` block that prints out `IncludeSourcePath:` and `IncludeTargetPath:`? See other `Verbose` logging in the file.